deploy-konflux.sh

```diff
@@ -36,6 +36,9 @@ deploy() {
 
     # The build-service depends on CRDs from the release-service
     retry kubectl apply -k "${script_path}/konflux-ci/build-service"
+    kubectl set --namespace build-service env \
+        deployment/build-service-controller-manager \
+        PAC_WEBHOOK_URL=http://pipelines-as-code-controller.pipelines-as-code.svc.cluster.local:8080
```

Comment:
Can we patch the deployment manifest instead?
I think it will be easier to track.

Reply:
Done, I submitted the change.